src/main/java/org/elasticsearch/index/mapper/core/AbstractFieldMapper.java

```diff
@@ -302,7 +302,7 @@ protected String buildFullName(BuilderContext context) {
     protected FieldDataType fieldDataType;
     protected final MultiFields multiFields;
     protected CopyTo copyTo;
-    protected final boolean writePre20Metadata;
+    protected final boolean writePre20Settings;
```

Comment:
Is it possible to name this something else? I see `writePre20Settings` and I think pre-0.20.0 version